ipaserver/install/installutils.py

```diff
@@ -1389,3 +1389,14 @@ def default_subject_base(realm_name):
 
 def default_ca_subject_dn(subject_base):
     return DN(('CN', 'Certificate Authority'), subject_base)
+
+
+def validate_mask():
+    try:
+        mask = os.umask(0)
+    finally:
+        os.umask(mask)
+    mask_str = None
+    if mask & 0b101101 > 0:
```

Comment:
Hi @fcami 
I am not sure I understand your binary mask here. If umask is set to 033, we still have read access but the validate_mask function will complain. Shouldn't we use 0b100100100 instead (to make sure we always keep read access)?

Reply:
Hi @flo-renaud 
Did the explanation seem valid to you? So that we can resolve this.

Reply:
Yes, so we accept masks composed of 0 or 2 but nothing else.